File: QViz.NET/Periscope.QViz/JSON/APIQueryValue.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Query Parameter Value Details
	/// </summary>
	public class APIQueryValue
	{
		/// <summary>
		/// Identifier of the API Query Value Object
		/// </summary>
		public string queryValueId;
		
		/// <summary>
		/// Key of the API HTTP Query String Object
		/// </summary>
		public string key;
		
		/// <summary>
		/// Value of the API HTTP Query String Object
		/// </summary>
		public string value;
		
		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;
		
		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIQueryValue()
		{
			this.queryValueId = "";
			this.key = "";
			this.value = "";
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIQueryValue.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Test API Query Parameter Value Details
 */
public class APIQueryValue {

	/**
	 * Identifier of the API Query Value Object
	 */
	public String queryValueId;

	/**
	 * Key of the API HTTP Query String Object
	 */
	public String key;

	/**
	 * Value of the API HTTP Query String Object
	 */
	public String value;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Default Constructor
	 */
	public APIQueryValue() {
		this.queryValueId = "";
		this.key = "";
		this.value = "";
		this.createdOn = null;
		this.updatedOn = null;
	}

}
